core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java

```diff
@@ -91,6 +94,8 @@
 @ThreadSafe
 public class FileSystemContext implements Closeable {
   private static final Logger LOG = LoggerFactory.getLogger(FileSystemContext.class);
+  private static final String TOTAL_THREAD_METRICS_NAME
+      = MetricsSystem.getMetricName(MetricKey.CLIENT_META_DATA_CACHE_SIZE.getName());
```

Comment:
Why assign the `CLIENT_META_DATA_CACHE_SIZE` to `TOTAL_THREAD_METRICS_NAME`, it make me confused?

Reply:
good catch, fixed, PTAL, thanks